### PR TITLE
MM-47093 : Migrate "components/integrations/installed_command.jsx" to Typescript

### PR DESCRIPTION
**Pull request by VishakhaPoonia**
#### Summary
Migrate "components/integrations/installed_command.jsx" to Typescript

#### Ticket Link
  Fixes https://github.com/mattermost/mattermost-server/issues/21076
  Jira: https://mattermost.atlassian.net/browse/MM-47093

#### Related Pull Requests
NA

#### Screenshots
NA

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Comment by hanzei** [comment]
@javaguirre Would you mind taking a look at this PR?

File: components/integrations/installed_command.test.tsx [hunk 1]
```diff
@@ -40,13 +42,13 @@ describe('components/integrations/InstalledCommand', () => {
...
     test('should match snapshot, not autocomplete, no display_name/description/auto_complete_hint', () => {
-        const minCommand = {
+        const minCommand = TestHelper.getCommandMock({
             id: 'r5tpgt4iepf45jt768jz84djic',
             trigger: 'trigger',
             auto_complete: false,
             token: 'testToken',
-            create_at: '1499722850203',
-        };
+            create_at: 1499722850203,
```
Comment (javaguirre) [inline components/integrations/installed_command.test.tsx:50]: The create_at changed from string to int, is that intended?

**Review by javaguirre — COMMENTED** [review]
Of course @hanzei ! I just have one question

File: components/integrations/installed_command.test.tsx (hunk 1 above)
Comment (VishakhaPoonia) [reply components/integrations/installed_command.test.tsx:50]: @javaguirre yes, as `type` of create_at is number, at least that’s what getting inferred

Comment (javaguirre) [reply components/integrations/installed_command.test.tsx:50]: In that case, I don't have anything else to add @VishakhaPoonia

I think there are conflicts, so if we resolved those I'm good with it. :-)

Thank you!

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
hey can you try rebasing with master once again i see lots of files

**Review by M-ZubairAhmed — APPROVED** [review]

**Comment by M-ZubairAhmed** [comment]
> Successfully triggered e2e testing! https://git.internal.mattermost.com/qa/cypress-ui-automation/-/pipelines/247146
QA report still stands valid, no significant changes were done after we ran this

**Review by javaguirre — APPROVED** [review]

**Review by jgilliam17 — APPROVED** [review]
Thanks @VishakhaPoonia 
Tested, looks good to merge.
[E2E report](https://automation-dashboard.vercel.app/cycles/e0514b3e-9d90-4d79-8902-fee7c367f0f7) - no PR related failures.

**Pull request merged**